windows/hardening/advancedAuditing.ps1

```diff
@@ -12,33 +12,43 @@ else {
 #This script enables auditing for various event categories on a Windows system
 
```

Comment:
Writing everything to the console is great. Does it also output to a log with timestamps? If not, that would be great for auditing and injects.

Reply:
Not actioning at this time.